site/content/entry/yuzu-progress-report-oct-2021/index.md

```diff
@@ -0,0 +1,430 @@
+
+Some games would crash if the user had a profile picture set in yuzu.
+[Narr](https://github.com/german77) discovered that this happened when the image's dimensions were greater than `256x256`.
+To avoid the crash, he made it so that [yuzu now resizes big profile pictures](https://github.com/yuzu-emu/yuzu/pull/7246) to fit the allowed dimensions instead.
+
+On Linux, opening the Configuration Window and pressing the `OK` button would lead consistently to a crash.
+[MightyCreak](https://github.com/MightyCreak) found the cause of this problem and submitted a patch to [solve it](https://github.com/yuzu-emu/yuzu/pull/7186).
+
+Another linux-only crash would happen when inputting text on `The Legend of Zelda: Breath of the Wild`.
+Morph investigated the problem and found it was caused by an out-of-bounds access in the string buffer, so 
+[he added the necessary checks](https://github.com/yuzu-emu/yuzu/pull/7172) to prevent it.
+
+## UI changes
+
+On the way to achieve better high DPI support, [Morph](https://github.com/Morph1984) [updated the version of Qt](https://github.com/yuzu-emu/yuzu/pull/7122) used by yuzu.
+This will allow in the near future to finally have legible UI elements on high DPI settings.
+
+MightyCreak has arrived with several internal changes, most helping with the build process of yuzu.
+One of them is [enabling mouse cursor auto-hide by default](https://github.com/yuzu-emu/yuzu/pull/7174), a very welcome quality of life change, specially now that more users 
+will be playing their games on fullscreen.
+Thanks!
+
+Linux gamepad users will no longer have to periodically tap their mouse to avoid the screensaver from taking over their screens! 
+toastUnlimited [properly disabled the screensaver while running yuzu](https://github.com/yuzu-emu/yuzu/pull/6702).
+
+[Moonlacer](https://github.com/Moonlacer) continues polishing the text strings on the UI, this time helping new users 
+[understand what our `NVDEC` (video decoding) options do](https://github.com/yuzu-emu/yuzu/pull/7165).
+
+Some users tend to think that NVDEC means that video decoding will only work on Nvidia GPUs, this is not the case.
+NVDEC is the name of the module in the Nintendo Switch, which uses the Tegra X1, an Nvidia SoC.
+This SoC uses the NVDEC module to decode videos, and the work yuzu has to do is emulate it to work on any compatible system, including systems without Nvidia GPUs on them. 
+We have a full explanation [here](https://yuzu-emu.org/entry/yuzu-nvdec-emulation/).
+
+So, for the sake of clarity, the options were renamed to more easily understand what’s the intended result.
+
+- `No Video Output` will result in a black screen every time a video wants to render, nothing is displayed, no fun.
+- `CPU Video Decoding` means processing videos using the CPU, this can be useful for AMD Windows users, since the AMD driver is notorious for having bad VP9 video decoding 
```

Comment:
```suggestion
- `CPU Video Decoding` means processing videos using the CPU. This can be useful for AMD Windows users, since the AMD driver is notorious for having bad VP9 video decoding 
```